widget/list.go

```diff
@@ -130,6 +130,8 @@ func (l *List) RefreshItem(id ListItemID) {
 		return
 	}
 	l.BaseWidget.Refresh()
+
+	l.Unselect(id)
```

Comment:
This seems like an unrelated change? Why are the semantics of RefreshItem changing to that you can't use it to refresh a selected item?

Reply:
But this code is not allowing an item to remain selected when it is refreshed - so scrolling will lose the selected status